src/Splunk.Logging.Common/HttpEventCollectorSender.cs

```diff
@@ -112,6 +114,7 @@ public enum SendMode
 
         private HttpClient httpClient = null;
         private HttpEventCollectorMiddleware middleware = null;
+        private Func<HttpEventCollectorEventInfo, dynamic> formatter = null;
```

Comment:
I would add a type for this Func. Look at what we did for HttpEventCollectorMiddleware.